src/ts/Html5Http.ts

```diff
@@ -1,28 +1,20 @@
-///<reference path="../ts/Conviva.d.ts"/>
-/*! (C) 2016 Conviva, Inc. All rights reserved. Confidential and proprietary. */
-/*! This is sample code meant to illustrate proper Conviva integration in video applications. */
-/*! This file should not be included in video applications as part of integrating Conviva. */
-
-// Implements Conviva.HttpInterface for Chrome.
-
 import HttpRequestCancelFunction = Conviva.HttpRequestCancelFunction;
 import HttpRequestCallback = Conviva.HttpRequestCallback;
 
 export class Html5Http implements Conviva.HttpInterface {
 
-  makeRequest(httpMethod: 'GET' | 'POST', url: string, data: string | null, contentType: string | null,
+  public makeRequest(httpMethod: 'GET' | 'POST', url: string, data: string | null, contentType: string | null,
               timeoutMs: number, callback: HttpRequestCallback | null): HttpRequestCancelFunction {
-    // XDomainRequest only exists in IE, and is IE8-IE9's way of making CORS requests.
-    // It is present in IE10 but won't work right.
-    // if (typeof XDomainRequest !== 'undefined' && navigator.userAgent.indexOf('MSIE 10') === -1) {
-    // 	return this.makeRequestIE89.apply(this, arguments);
-    // }
```

Comment:
This comment, as the others below, are part of the Conviva SDK source files so please keep it in place.

Reply:
@stonko1994 https://github.com/bitmovin/bitmovin-player-analytics-conviva/pull/9#discussion_r241367837